ocs_ci/utility/ibmcloud.py

```diff
@@ -203,3 +205,193 @@ def add_deployment_dependencies():
     ]
     for cr in wa_crs:
         run_cmd(f"oc apply -f {cr}")
+
+class IBMCloud(object):
+    """
+    Wrapper for Ibm cloud
+    """
+
+    def __init__(self):
+        """
+        Class Initialization.
+        """
+        logger.info(f"ibmcloud class")
+
+
+    def restart_nodes(self, nodes, timeout=900, wait=True):
```

Comment:
As I pointed out if the nodes is list and function is called restart_nodes I guess you should iterate over the nodes and restart all in the list. Or change the function to take only one node instead of list and rename function to restart_node